src/components/task-plan/homework.cjsx

```diff
@@ -68,6 +68,7 @@ ChooseExercises = React.createClass
         hide={hide} />
 
       <PinnedHeaderFooterCard
+        containerBuffer={50}
```

Comment:
Could/Should this be a var we refer to in other places as well? And make it required? That way the size is explicitly described?